.example.env

```diff
@@ -23,12 +23,6 @@ RESOLVER=127.0.0.11
 HTTPS_HOST=${COMPUTER_HOSTNAME}
 HTTPS_PORT=443
 
-# define media and static url
-MEDIA_ROOT=/django_project/minisass/static
```

Comment:
I think for the testing site we still need this to be set to `MEDIA_ROOT=/home/web/django_project/minisass/static/`

Reply:
@NyakudyaA alright I have made the modifications